.github/workflows/integrations-updater.yml

```diff
@@ -55,7 +55,11 @@ jobs:
       - name: Publish to meshery.io site
         working-directory: ./meshery/mesheryctl
         run: ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw ../../meshery.io/integrations ../meshery.io/assets/images/integration -o js
-
+      
+      - name: Publish OCI images to meshery.io website
+        working-directory: ./meshery/mesheryctl
+        run: ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw ../../meshery.io/integrations ../meshery.io/assets/images/integration -o oci
```

Comment:
According to `mesheryctl registry publish --help`, the supported output formats are:

```
  -o, --output-format string   output format [md | mdx | js]
```